test/test_main.cpp

```diff
@@ -404,6 +404,29 @@ void RelatedGL(const Manifold& out, const std::vector<MeshGL>& originals,
   }
 }
 
+void ExpectProperties(const MeshGL& mesh, int numProp,
```

Comment:
I don't think this function is necessary, as I believe `RelatedGL` already checks all of this in a more generic way.

Reply:
OK, removed